### ISSUE TITLE
Tabs do not fit screen / Are hidden

### ISSUE THREAD

**Issue by dkwamboka** [onaio/gisida#310]
Ideally, when you click on the Summary Page, the Chart Tabs should fit/ show fully regardless of your machine's screen size as below:
<img width="960" alt="ideal" src="https://user-images.githubusercontent.com/20725261/60176842-1b8b4c00-9820-11e9-9012-cd28f7371275.png">

Currently, his isn't the case, some tabs are not visible:

<img width="960" alt="currently" src="https://user-images.githubusercontent.com/20725261/60176881-35c52a00-9820-11e9-9ded-b565f4ada801.png">

**Comment by cKellyDesign** [comment]
We can either do a pagination of the tabs (medium LOE), or a dropdown with overflow tabs (low LOE).

@rowo @OluwatobiMayowa - Thoughts on this? Mockups or wireframes would be helpful too if possible!

**Comment by rowo** [comment]
@dkwamboka Those tab labels seem like they can be shortened so I'd recommend that regardless.

@cKellyDesign when you say dropdown with overflow tabs are you talking about something like https://www.inventpartners.com/open-source/bootstrap-responsive-tabs ? I think this is the way to go if so.

**Comment by cKellyDesign** [comment]
@rowo Yeah that looks like exactly what I was thinking, I think this [react-responsive-tabs module](https://github.com/maslianok/react-responsive-tabs) would integrate quite nicely!

@KipSigei @kahama94 @moshthepitt - what do you think?

**Comment by cKellyDesign** [comment]
@KipSigei What's the status of this?
